bundles/org.openhab.ui.basic/web-src/_layout.scss

```diff
@@ -218,6 +218,22 @@
 			box-shadow: none;
 			-webkit-box-shadow: none;
 		}
+		.mdl-textfield {
+			&__input,
+			&__label {
+				font-weight: normal;
+				font-size: 14px;
```

Comment:
I see that the font is different than the one for Text widget. I was asking myself if we should not use the same ?
And the size should be the same, so to be checked.

Reply:
Font and font size should be the same, only the weight (not bold). I did it on purpose, but will change. I will still keep the placeholder without bold, again to differentiate clearly.